packages/genome-nexus-ts-api-client/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "genome-nexus-ts-api-client",
+  "description": "Genome Nexus API Client for TypeScript",
+  "version": "0.1.0",
```

Comment:
This version should reflect the actual API version, ideally the latest tag version on genome-nexus master branch.

Reply:
setting this to `1.1.2-beta.0` for now. latest master is 17 commits ahead of the latest release `1.1.1` so it is better set it to the next beta version.